controllers/topology/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - cluster-api-topology-maintainers
+
+reviewers:
+  - cluster-api-reviewers
```

Comment:
I duplicated one of our existing files. To be honest I'm not sure why we're adding `cluster-api-reviewers` here again. I would have assumed it's not necessary (similar to cluster-api-maintainers for approvers)

Reply:
Since cluster-api-maintainers and cluster-api-reviewers are present in the top level OWNERS file we technically dont need them here. Anyone in the root owners file can approve anything in the project.

Reply:
I don't think reviewers actually are inherited, vs approvers